src/Glue.elm

```diff
@@ -71,6 +71,52 @@ glue =
     Glue
 
 
+{-| Create [Glue](#Glue) mapigs between modules.
+child module can be generic TEA app or module that is already doing polymorfic maping to generic `msg` internaly.
+You can also use `Cmd` for sending data from bottom module to upper one if you want to observe child
+as a black box (similary you do in case of DOM events with `Html.Events`).
+
+**Interface**:
+
+    createglue :
+          { modelsetter = \subModel model -> { model | subModel = subModel }
+          , modelgetter = .subModel
+          , init =  ( subModel, Cmd subMsg )
+          , update = subMsg -> subModel -> ( subModel, Cmd msg )
+          , view = subModel -> Html subMsg
+          , subscriptions = subModel -> Sub subMsg
+          , liftmessage = subMsg -> msg
+          }
+        -> Glue model subModel msg subMsg
+
+See [examples](https://github.com/turboMaCk/glue/tree/master/examples) for more informations.
+
+-}
+createglue :
+    { e
+        | init : ( subModel, Cmd subMsg )
+        , liftmessage : subMsg -> msg
```

Comment:
You mean change liftmessage to msg?
We could do that but wouldn't it be confusing for newbies with the two types subMsg and msg and also a field called msg?  

According to https://guide.elm-lang.org/types/union_types.html
each message of a union type is a value.
![image](https://cloud.githubusercontent.com/assets/6956837/25729209/212216f0-30f1-11e7-8c4a-f44cfcf9fa87.png)

How about msgValue instead?